### PR TITLE
add xinabox OD01 and breakout display section

### PR DESCRIPTION
**Pull request by microbit-mark**
cc @luqmaanb @PragmaticPhil

### CODE REVIEW

**Comment by microbit-mark** [comment]
please follow up on this one @abchatra @pelikhan

**Comment by PragmaticPhil** [comment]
Thanks Mark :)
@pelikhan / @abchatra - please let me know if there is anything I can do to help this one progress.

File: docs/extensions.md [hunk 1]
```diff
@@ -166,6 +166,26 @@ Check out [microbit.org](http://microbit.org/resellers/) for more information on
 }]
 ```
 
+## Display
+
+```codecard
+[{
+  "name": "XinaBox OD01 Display",
+  "url":"/pkg/inabox/pxt-OD01",
```
Comment (pelikhan) [inline docs/extensions.md]: missing x

**Comment by pelikhan** [comment]
is this a clone of https://github.com/microsoft/pxt-microbit/pull/2390/files?

File: docs/extensions.md (hunk 1 above)
Comment (pelikhan) [reply docs/extensions.md]: @microbit-mark

**Comment by PragmaticPhil** [comment]
Please let us know if there anything we can do to make progress on this.

**Comment by abchatra** [comment]
xinabox/pxt-OD01 is also missing MIT license.

File: docs/extensions.md (hunk 1 above)
Comment (pelikhan) [reply docs/extensions.md]: @microbit-mark

**Comment by PragmaticPhil** [comment]
@abchatra Thank you for the pointer on this - the license has been added :)

File: docs/extensions.md (hunk 1 above)
Comment (microbit-mark) [reply docs/extensions.md]: Thank @peli. Been on leave, but have fixed this now

**Comment by abchatra** [comment]
@pelikhan approved from my side. Can you merge if your comments are addressed?

**Comment by pelikhan** [comment]
Please answer https://github.com/microsoft/pxt-microbit/pull/2397#issuecomment-529966186

**Comment by PragmaticPhil** [comment]
@pelikhan - sorry, I somehow missed your question on clones. I am not sure if it was cloned from that library some time ago - @shaoziyang worked on both so it is possible (maybe likely) they have a common root.  Right now they are very different though - we've implemented a lot of changes since we forked from Shao.

File: targetconfig.json [hunk 2]
```diff
@@ -140,7 +140,7 @@
             "sparkfun/pxt-gator-soil",
             "sparkfun/pxt-gator-microphone",
             "rebeccaclavier/pxt-bmp280",
-            "makecode-extensions/HTS221",
```
Comment (pelikhan) [inline targetconfig.json]: looks like a merge issue

**Pull request merged**